doc/admin-guide/plugins/cache_range_requests.en.rst

```diff
@@ -158,13 +127,25 @@ to this header.
 
 .. code::
 
-    X-Crr-Ident: Etag: "foo"
-    X-Crr-Ident: Last-Modified: Tue, 19 Nov 2019 13:26:45 GMT
+    X-Crr-Ident: Etag "foo"
```

Comment:
Is it obvious that this is the Ident order of which they are checked? If not, maybe it would be helpful to state that so even if Stale is explicitly used, it won't be used if Etag is used